src/content/configuration/externals.md

```diff
@@ -11,14 +11,15 @@ contributors:
   - Mistyyyy
   - jamesgeorge007
   - tanhauhau
+  - snitin315
 ---
 
 The `externals` configuration option provides a way of excluding dependencies from the output bundles. Instead, the created bundle relies on that dependency to be present in the consumer's (any end-user application) environment. This feature is typically most useful to __library developers__, however there are a variety of applications for it.
 
 
 ## `externals`
 
-`string` `object` `function`  `RegExp`
+`string` `array` `object` `function`  `RegExp`
```

Comment:
We are in a habit of specifiyng what's allowed as array value, therefore i suggest going for:

```suggestion
`string` `[string, RegExp, function => string]` `object` `function => string`  `RegExp`
```